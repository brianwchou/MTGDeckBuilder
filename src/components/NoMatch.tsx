import React from 'react'
import {Link} from 'react-router-dom'

export default function NoMatch () {
    return (
        <div>
            <h1>link does not exist</h1>
            <Link to = {'/home'}>click to go back to Home page</Link> 
        </div>
    )
}