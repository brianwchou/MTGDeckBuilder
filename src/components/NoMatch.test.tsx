import React from 'react';
import NoMatch from '../components/NoMatch'
import renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom';

describe('NoMatch', () => {

    test('renders correctly', () => {
        const tree = renderer
            .create(<Router><NoMatch /></Router>)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
})