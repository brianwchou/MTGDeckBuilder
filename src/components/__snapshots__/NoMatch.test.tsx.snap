// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch renders correctly 1`] = `
<div>
  <h1>
    link does not exist
  </h1>
  <a
    href="/Home"
    onClick={[Function]}
  >
    click to go back to Home page
  </a>
</div>
`;
